hazelcast/src/test/java/com/hazelcast/test/AbstractHazelcastClassRunner.java

```diff
@@ -64,9 +65,10 @@
     static {
         String loggingType = "hazelcast.logging.type";
         String loggingClass = "hazelcast.logging.class";
-        if (System.getProperty(loggingType) == null && System.getProperty(loggingClass) == null) {
+        if (JenkinsDetector.isOnJenkins()) {
```

Comment:
Good idea. 

I see the `JenkinsDetector.isOnJenkins()` as a migration measure. Once/if the code is merged then we should remove the logging configuration from Jenkins builds. It's duplicated on way too many places (in all builds)